scripts/configure.py

```diff
@@ -159,6 +159,14 @@
   'lms' : {
     'apt': ['libcrypt-openssl-rsa-perl'], # needed for ShairTunes2W support
     'copy' : [{'from': 'bin/ARCH/squeezelite', 'to': 'streams/squeezelite'}],
+    'script' : [
```

Comment:
Should this upgrade older versions?

Reply:
Its really fine if it doesnt for now, but we may want to add support for this in the next release 0.3.1?.

Reply:
I think we should manage a full-blown repository moving forward, which will address this. There are a few cases in the world where not upgrading versions is plainly a problem, but those cases are also aware that they need to reflash to get the bugfixes in latest LMS anyhow (and the 0.3.0 image we produce will have this version.)